scylladb/sphinx-scylladb-theme
# Add configurable banner

Related issue https://github.com/scylladb/scylla-docs/issues/3554

This PR will allow projects to set up their own custom banners for announcements.

## How to test this PR

1. In conf.py -> html_options, set the following options:

```
    html_theme_options = {
        ...
        'hide_banner': 'false',
        'banner_title_text': 'Sample Title Text',
        'banner_button_text': 'Learn More >',
        'banner_button_url': 'https://university.scylladb.com/',
    }
````

2. Run ``make preview``. The banner button should redirect you to https://university.scylladb.com/.

## What's next?

After we release a new version of the theme with this functionality, I'll issue a PR in scylla-docs setting up the banner for the next Scylla University Live event. Then, you should be able to change the text and link yourself with the new options added.
